multimodal/setup.py

```diff
@@ -65,7 +65,7 @@
     "onnx>=1.13.0,<1.14.0",
     "onnxruntime>=1.15.0,<1.16.0;platform_system=='Darwin'",
     "onnxruntime-gpu>=1.15.0,<1.16.0;platform_system!='Darwin'",
-    "tensorrt>=8.5.3.1,<8.5.4;platform_system=='Linux' and python_version<'3.11'",  # tensorrt > 8.5.4 cause segfault
+    "tensorrt>=8.6.0,<8.6.2;platform_system=='Linux' and python_version<'3.11'",
```

Comment:
I think we can drop the python 3.11 cap here as well, now that we upgrade tensorrt.

py3.11, tensorrt==8.6.1 and cuda==12.1 is a combination that worked for me before, as discussed offline yesterday @prateekdesai04 .

Reply:
I kept it for now because dropping python 3.11 fails some of the HPO tests.
A good route would be merging the current PR and then removing python 3.11 in another PR as there is a lot of dependencies at play specifically in this PR, and it would also unblock the release items.